src/schemas/json/aurora-1.0.json

```
@@ -0,0 +1,295 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "definitions": {
+    "property": {
+      "type": "object",
+      "required": ["name", "type"],
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of property, in camelCase"
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of property",
+          "enum": [
+            "bigint.unsigned",
+            "bigint",
+            "blob.long",
+            "blob.medium",
+            "blob.tiny",
+            "blob",
+            "boolean",
+            "char",
+            "date",
+            "decimal",
+            "enum",
+            "float",
+            "id",
+            "int.unsigned",
+            "int",
+            "json",
+            "manyToMany",
+            "password",
+            "relationship",
+            "smallint.unsigned",
+            "smallint",
+            "text",
+            "timestamp",
+            "tinyint.unsigned",
+            "tinyint",
+            "varchar"
+          ]
+        },
+        "length": {
+          "type": "number",
+          "description": "Set max length to property"
+        },
+        "maxLength": {
+          "type": "number",
+          "description": "Set max length to property"
+        },
+        "nullable": {
+          "type": "boolean",
+          "description": "Set property to nullable"
+        },
+        "primaryKey": {
+          "type": "boolean",
+          "description": "Set property to primary key"
+        },
+        "index": {
+          "type": "string",
+          "description": "To define property like a index",
+          "enum": ["index", "unique"]
+        },
+        "indexName": {
+          "type": "string",
+          "description": "The name of index, if there are various indexes with the same name, they will be grouped"
```

```suggestion
          "description": "The name of index, if there are various indexes with the same name, they will be grouped"
```

If name can't be an empty string use `minLength` constraint.

---

Thanks! I have fixed it in the PR #3018